src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,56 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private static final int NOT_FOUND_INDEX = -1;
+    private int size = 0;
```

Comment:
0 is the default value for primitive int fields
```suggestion
    private int size;
```